KubeArmor/policy/policy.go

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package policy
+
+import (
+	"context"
+	"encoding/json"
+
+	kg "github.com/kubearmor/KubeArmor/KubeArmor/log"
+	tp "github.com/kubearmor/KubeArmor/KubeArmor/types"
+	pb "github.com/kubearmor/KubeArmor/protobuf"
+)
+
+// PolicyServer Structure
+type PolicyServiceServer struct {
+	pb.PolicyServiceServer
```

Comment:
@daemon1024 In this struct for server, please check any references. As per my understanding we need to use **pb.UnimplementedPolicyServiceServer**

This might throw an error later when establishing connection

Reply:
seems to be working fine, in it's current state.